src/main/java/ru/hogwarts/school/controller/FacultyController.java

```diff
@@ -1,37 +1,45 @@
 package ru.hogwarts.school.controller;
 
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 import ru.hogwarts.school.model.Faculty;
-import ru.hogwarts.school.service.FacultyService;
+import ru.hogwarts.school.service.FacultyServiceImpl;
 
 import java.util.List;
 
 @RestController
-@RequestMapping("faculty")
+@RequestMapping("/faculty")
 public class FacultyController {
-    private final FacultyService service;
+    private final FacultyServiceImpl service;
 
-    public FacultyController(FacultyService service) {
+    public FacultyController(FacultyServiceImpl service) {
```

Comment:
```suggestion
    public FacultyController(FacultyService service) {
```
инъекцию надо делать по интерфейсам, а не реализациям, чтобы хоть где-то был смысл в использовании интерфейсов для сервисов